api/src/main/java/uk/gov/dhsc/htbhf/claimant/scheduler/PaymentCycleScheduler.java

```diff
@@ -45,7 +45,18 @@ public void createNewPaymentCycles() {
         List<ClosingPaymentCycle> cycles = paymentCycleRepository.findActiveClaimsWithCycleEndingOnOrBefore(cycleEndDate);
 
         log.info("Creating new PaymentCycles for {} claims", cycles.size());
-        cycles.forEach(cycle -> job.createNewPaymentCycle(cycle.getClaimId(), cycle.getCycleId(), cycle.getCycleEndDate().plusDays(1)));
+        cycles.forEach(cycle -> createNewPaymentCycle(cycle));
         log.debug("Finished creating new PaymentCycles for {} claims", cycles.size());
     }
+
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
```

Comment:
This is looking like a candidate for removing this warning, it is appearing a good number of times now, I don't have too much objection to catching generic Exceptions, you often need to do this at a high level in your application as we are doing.